include/ppx/grfx/grfx_command.h

```diff
@@ -162,6 +162,40 @@ struct ImageToImageCopyInfo
     } extent;
 };
 
+//! @struct ImageBlitInfo
+//!
+//!
+struct ImageBlitInfo
+{
+    struct
```

Comment:
srcImage and dskImage use the same struct so that they can be only declared once instead of twice?